Duke.java

```diff
@@ -0,0 +1,185 @@
+import java.util.Scanner;
+
+public class Duke {
+    // static variables
+    private static Task[] taskList = new Task[100];
```

Comment:
This code violates the coding standards. As mentioned on the module website: 

> Plural form should be used on names representing a collection of objects.

As such, perhaps you can consider using a more appropriate name (e.g. `tasks`) to comply with the coding standards.